src/g_game.c

```diff
@@ -1496,6 +1496,9 @@ static void G_DoCompleted(void)
   if (automapactive)
     AM_Stop();
 
+  // Rebuild the Time widget to get rid of the Use-button timer
```

Comment:
Shouldn't this get called before `gameaction = ga_nothing`?

Reply:
Or you could just reset `plr->btuse_tics = 0` in `G_PlayerFinishLevel()` (where all the other level-specific properties of `struct player_t` are reset) and get rid of that check in `HU_widget_build_sttime()`.

Reply:
Good idea! It seems to work; done in d31a756f0b572c463cc833fd6cf129d996cfc8a6.